apps/frontend/src/components/global/ExportASFFModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <v-dialog v-model="showingModal" width="500px">
+    <template #activator="{on}">
+      <LinkItem
+        key="export_csv"
+        text="Export as ASFF"
+        icon="mdi-file-document-multiple"
+        @click="showModal"
+        v-on="on"
+      />
+    </template>
+    <v-card>
+      <v-card-title class="headline"> Export as ASFF </v-card-title>
+      <v-card-text>
+        <v-text-field v-model="awsAccountId" label="AWS Account ID" />
+        <v-text-field v-model="target" label="Target Name" />
+        <v-text-field v-model="region" label="Region" />
```

Comment:
Can we provide an information icon letting the user known what he/she needs to provide for each field. For example:
AWS Account ID -> To find out your ASW account ID, log into AWS, in the navigation bar, choose Support, and then Support Center. Your currently signed-in 12-digit account number (ID) appears in the Support Center navigation pane.
Target Name -> II believe it is the name they want to see in AWS, but list here the correct message
Region -> To find out your ASW Region, log into AWS, and look at the top bar. See [AWS Regions](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html)